src/manager/transactions/update-contract.cdc

```diff
@@ -0,0 +1,21 @@
+import FlowManager from 0x01
+
+transaction(name:String, code: String, manager: Address ##ARGS-WITH-TYPES##) {
+    prepare(acct: AuthAccount){
+        let decoded = code.decodeHex()
+
+        if acct.contracts.get(name: name) == nil {
+          acct.contracts.add(name: name, code: decoded)
+        } else {
+          acct.contracts.update__experimental(name: name, code: decoded)
+        }
+
+        let linkPath = FlowManager.linkContractManager
```

Comment:
Consider renaming this `linkContractManager` is somewhat unclear on first reading.